test/evergreen.yml

```diff
@@ -2640,7 +2587,59 @@ buildvariants:
       LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libeatmydata.so PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
   tasks:
     - name: ".pull_request !.windows_only !.pull_request_compilers !.python"
-    - examples-c-test
+    - name: make-check-test
```

Comment:
The make check tests actually had been split into individual tasks and covered by the previous line i.e. `.pull_request` tag selector, in order to control the make span of this required builder.

Reply:
Thanks! I'll remove the task.